### DISCUSSION TITLE
Custom project templates

### DISCUSSION THREAD

**Discussion by stephenlutes** [wagtail/wagtail#9982] [Ideas]
Is there a chance that custom project templates could be supported. It should take minimal changes, mainly allowing a path to the custom template be given in the start command.

**Comment by gasman** [comment]
The `wagtail start` command is really just `django-admin startproject` with its own custom template passed in, so `wagtail start myproject --template=/some/template/path` would be exactly functionally equivalent to `django-admin startproject myproject --template=/some/template/path`, I think. Is there a benefit to it existing as part of the `wagtail start` command too? (Familiarity, I guess...)

**Reply by stephenlutes** [reply]
I saw that it was a part of the django-admin startproject, but for some reason it didn't register to just pass the argument. I don't think there is additional benefit to adding it to the wagtail start command.
